### PR TITLE
Blerg

### PR DESCRIPTION
**Pull request by chrismccord**

### CODE REVIEW

File: source/posts/2016-08-09-phoenix-channels-vs-rails-action-cable.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+Now that Rails 5.0 has been released, we have clients with existing Rails stacks asking if they should use Action Cable for their real-time features, or jump to Phoenix for more reliable and scalable applications. When planning architecture decisions, in any language or framework, you should always take measurements to prove out your assumptions and needs. To measure how Phoenix channels and Action Cable handle typical application work-flows, we created a chat application in both frameworks and stressed each through varying levels of workloads.
+
+## How the tests were designed
+
+For our measurements, we used the [tsung](http://tsung.erlang-projects.org) benchmarking client to open WebSocket connections to each application. We added XML configuration to send specific Phoenix and Rails protocol messages to open channels on the established connection and publish messages. For hardware, we used two Digital Ocean 16GB, 8-core instances for the client and server.
+
+The work-flows for each tsung client connection were as follows:
+
+- open a WebSocket connection to the server
+- create a single a channel subscription on the connection, to a chat room chosen at random
```
Comment (bcardarella) [inline source/posts/2016-08-09-phoenix-channels-vs-rails-action-cable.md]: "create a single a channel" => "create a single channel"

File: source/posts/2016-08-09-phoenix-channels-vs-rails-action-cable.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+## Rails Memory Leaks
+
+After creating the Rails chat application, setting up redis, and deploying the application to our instance, we immediately observed a memory leak in the application that was visible just by refreshing a browser tab and watching the memory grow; to never be freed. The following recording shows this in action (sped up 10x):
+
+![](http://i.imgur.com/jQOIhkX.gif)
+
+We searched recently reported bugs around this area, and found an issue related to Action Cable [failing to call `socket.close` when cleaning up connections](https://github.com/rails/rails/pull/25615/files). This patch has been applied to the `5-0-stable` branch, so we updated the app to the unreleased branch and re-ran the tests. The memory leak persisted.
+
+We haven't yet isolated the source of the leak, but given the simplicity of the channel code, it must be within the Action Cable stack. This leak is particularly concerning since Rails 5.0 has been released for some time now and the `5-0-stable` branch itself has unreleased memory leak patches going back greater than 30 days.
```
Comment (bcardarella) [inline source/posts/2016-08-09-phoenix-channels-vs-rails-action-cable.md]: :trollface:

**Comment by bcardarella** [comment]
Very nice :)

File: source/posts/2016-08-09-phoenix-channels-vs-rails-action-cable.md (hunk 2 above)
Comment (rafaelfranca) [inline source/posts/2016-08-09-phoenix-channels-vs-rails-action-cable.md]: :smile: very good behavior, as expected. Do you want to release? I can give access 😉

**Pull request merged**